content/articles/laravel-8-collections/index.md

```diff
@@ -0,0 +1,251 @@
+### Introduction
+
+Laravel framework has several features easing the development process.   
+In this tutorial, I'll show you how to use Laravel collections, which work just like PHP arrays, although with additional wrapper features. We'll see how to use them in our codes with a simple demonstration project.  
+
+### Table of contents
+
+- [Introduction](#introduction)
+- [Table of contents](#table-of-contents)
+- [Prerequisites](#prerequisites)
+- [Objectives](#objectives)
+- [What is a Laravel collection?](#what-is-a-laravel-collection)
+- [Creating Laravel collections](#creating-laravel-collections)
+- [Eloquent ORM Collections](#eloquent-orm-collections)
+- [Finding data](#finding-data)
+
+### Prerequisites
+
+- This tutorial requires the reader to create a Laravel project to follow along. Additionally, you should have basic knowledge of Laravel, especially routing and controllers.
+
+### Objectives
+
+This tutorial teaches you everything you need to know about Laravel collections. By the end, you should have substantial know-how to get you started.
+
+### What is a Laravel collection?
+
+It's a modified class version of PHP arrays located in the ` Illuminate\Support\Collection ` providing a wrapper to work with arrays of data.  
+
+Consider the following code snippet:  
+
+```php
+//creating a test for collection for insights
+public function collect(){
+
+    //return the collection in uppercase
+    return collect(['john', 'doe', null])->map(function ($test_name) {
+        return strtoupper($test_name);
+    })->reject(function ($test_name) {
+        return empty($test_name);
+    });
+}
+```
+
+Output:
+
+```bash
+["JOHN", "DOE"] # output displayed on the browser
+```
+
+In the above snippet, we use the `collect()` method to create a Collection instance from the defined array modifying each element to uppercase then removing empty elements. We can as well see that this class allows us to chain different methods to achieve our objective.
+
```

Comment:
@WanjaMIKE, every single code snippet has an explanation at the end : Look at the example above, it explains what happens the `collect()` method that has been used in the code!